docs/src/main/paradox/connectors.md

```diff
@@ -9,6 +9,7 @@
 * [Cassandra Connector](cassandra.md)
 * [File Connectors](file.md)
 * [FTP Connector](ftp.md)
+* [Google PubSub Connector](googlepubsub.md)
```

Comment:
Should we call it Google Cloud PubSub? This is how it shows in their docs https://cloud.google.com/pubsub/